apps/site/lib/site_web/views/trip_plan_view.ex

```diff
@@ -523,6 +523,47 @@ defmodule SiteWeb.TripPlanView do
     SiteWeb.ViewHelpers.mode_name(type)
   end
 
+  @spec transfer_note(Itinerary.t()) :: String.t() | nil
+  def transfer_note(itinerary) do
+    itinerary
+    |> Itinerary.route_ids()
+    |> Enum.chunk_every(2, 1, :discard)
+    |> Enum.find(&satisfies_transfer_note_conditions?(&1))
```

Comment:
Based on our conversation this morning it sounds like we now want to display the note in all cases. I don't know if you want to remove this logic in this PR, or just wait and do it later. Either way seems fine to me.

Reply:
I don't think I'd say **all**; for example for single-route trips it wouldn't make sense. We'll leave it for later..

Reply:
Good point